packages/components/src/components/Heading/index.ts

```diff
@@ -59,6 +59,12 @@ export const HeadingComponentDefinition: ComponentDefinition = {
       description: 'The text bold of the heading.',
       defaultValue: true,
     },
+    cfBackgroundColor: {
+      displayName: 'Background color',
+      type: 'Text',
+      group: 'style',
+      description: 'The background color of the heading',
+    },
```

Comment:
I don't think this addition is needed.  This variable is inherited from `builtInStyles` const exported from packages/core/src/definitions/styles.ts, so we would only need to define this variable here if we wanted to set a different `defaultValue`.